### ISSUE TITLE
Translation: "Zum Seitenanfang" in the default English docs page

### ISSUE THREAD

**Issue by eyedean** [nodejs/nodejs.org#2267]
Hi,

Please check out https://prnt.sc/o3kk97

On https://nodejs.org/de/docs/guides/debugging-getting-started/ when I scroll down half the document and scroll up a little bit, I see a "Zum Seitenanfang" bubble at the bottom right of the screen.  Using Google Translate, I found it is "to the top" (which is also the `id` of the container `<a>` tag for that.)

It's happening on both Chrome and Firefox.

PS. As someone who had worked on multilingual products, I've seen such translation issues usually happen when the translation package is not received from volunteers/vendors yet.  So, I am wondering if Node.js's primary language is German. ;D (Cheers to all EU contributors!)

**Comment by Trott** [comment]
> https://nodejs.org/de/docs/guides/debugging-getting-started/

@eyedean You're looking at the German page. (See the `de` in the URL?) The English version is https://nodejs.org/en/docs/guides/debugging-getting-started/ where that bubble says "Scroll to top". 

Unfortunately the "German" page is mostly in English because nobody has translated this page into German yet....

**Comment by eyedean** [comment]
Hah, great catch!  

I got there by googling "Nod Debug Chrome".  It's interesting that the `/de/` version is the only result from nodejs.org that shows up in Google (second result).  I tried in incognito too, and it was the same.

I tried to compare the HTML [source code of `/de`](view-source:https://nodejs.org/de/docs/guides/debugging-getting-started/) with [the `/en` version](view-source:https://nodejs.org/en/docs/guides/debugging-getting-started/) and the only difference I found was the following line being only in `/de` version:
```html
  <link rel="alternate" href="/de/feed/tsc-minutes.xml" title="Node.js TSC-Sitzungsprotokoll" type="application/rss+xml">
```

Maybe someone else can say what has caused Google to rank `/de` over `/en`, and is it something we can fix or not.

**Comment by Trott** [comment]
Easiest solution might be to see if someone might volunteer to translate that page to German so it shows up in German search results and not so much in English ones. :-D 

/ping @nodejs/nodejs-de

**Comment by fhemberger** [comment]
The page doesn't exist in German. So at build time, it's generated using the English content and the translated page frame. We switched to this rather than showing users a 404 page or redirecting them to the English version, which might be confusing as they'll continue to browse the site in English.

The page uses the correct `<html lang="de">`, so this seems to be a Google ranking issue.

But I removed the dead link to the TSC minutes RSS feed.
